README.md

```diff
@@ -28,51 +35,53 @@ to support GPU inference in the future through alternate backends.
 
 Currently, the following models are supported:
 
+- [BLOOM](https://huggingface.co/docs/transformers/model_doc/bloom)
 - [GPT-2](https://huggingface.co/docs/transformers/model_doc/gpt2)
 - [GPT-J](https://huggingface.co/docs/transformers/model_doc/gptj)
-- [LLaMA](https://huggingface.co/docs/transformers/model_doc/llama): LLaMA,
-  Alpaca, Vicuna, Koala, GPT4All v1, GPT4-X, Wizard
 - [GPT-NeoX](https://huggingface.co/docs/transformers/model_doc/gpt_neox):
   GPT-NeoX, StableLM, RedPajama, Dolly v2
-- [BLOOM](https://huggingface.co/docs/transformers/model_doc/bloom): BLOOMZ
+- [LLaMA](https://huggingface.co/docs/transformers/model_doc/llama): LLaMA,
+  Alpaca, Vicuna, Koala, GPT4All v1, GPT4-X, Wizard
 - [MPT](https://www.mosaicml.com/blog/mpt-7b)
 
-## Getting started
+## Getting Started
 
-Make sure you have a Rust 1.65.0 or above and C toolchain[^1] set up.
+This project depends on Rust v1.65.0 or above and a modern C toolchain.
 
-`llm` is a Rust library that re-exports `llm-base` and the model crates (e.g.
-`bloom`, `gpt2` `llama`).
+The `llm` crate exports `llm-base` and the model crates (e.g. `bloom`, `gpt2`
+`llama`).
 
 To use `llm`, add it to your `Cargo.toml`:
 
 ```toml
 [dependencies]
-llm = "0.1"
+llm = "0.2"
 ```
 
-**NOTE**: For best results, make sure to build and run in release mode. Debug
-builds are going to be very slow.
+**NOTE**: To improve debug performance, exclude `llm` from being built in debug
+mode:
+
+```toml
+[profile.dev.package.llm]
+opt-level = 3
+```
 
 ### Building `llm-cli`
 
-`llm-cli` (binary name `llm`) is a basic application that provides a CLI
-interface to the library.
+Follow these steps to build the command line application, which is named `llm`:
```

Comment:
command line -> command-line

We should also mention downloading the CLI from the GitHub releases.